### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by almaslennikov**

### CODE REVIEW

**Review by e0ne — APPROVED** [review]

File: .github/workflows/image-push-master.yaml [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    - name: login to GitHub Container Registry
+      uses: docker/login-action@v2
+      with:
+        registry: ghcr.io
+        username: ${{ github.repository_owner }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Create manifest for multi-arch images
+      run: |
+        docker buildx imagetools create -t ${{ env.IMAGE_NAME }}:latest -t ${{ env.IMAGE_NAME }}:sha-${{ github.sha }} \
```
Comment (ykulazhenkov) [inline .github/workflows/image-push-master.yaml:114]: In this repo existing  tags with commit hash are without `sha-` prefix

File: .gitignore [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # Files
 .gopath
+.cover
 
 # Folders
 build/
 bin/
+vendor/
```
Comment (ykulazhenkov) [inline .gitignore:8]: no new line at the end

File: .github/workflows/image-push-master.yaml (hunk 1 above)
Comment (almaslennikov) [reply .github/workflows/image-push-master.yaml]: Thanks for catching!

File: Dockerfile [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
 
 COPY . /usr/src/ib-sriov-cni
 
 ENV HTTP_PROXY $http_proxy
 ENV HTTPS_PROXY $https_proxy
 
-RUN apk add --update --virtual build-dependencies build-base linux-headers git && \
-    cd /usr/src/ib-sriov-cni && \
-    make clean && \
+RUN apk add --no-cache --virtual build-dependencies build-base=~0.5 linux-headers=~6.3
```
Comment (e0ne) [inline Dockerfile:8]: why do we need to specify minimum version here?

Comment (almaslennikov) [reply Dockerfile:8]: It's a requirement for hadolint check

**Review by ykulazhenkov — APPROVED** [review]

**Review by adrianchiris — APPROVED** [review]

**Pull request merged**